src/schema/objects/metadata.yaml

```
@@ -901,7 +901,7 @@ DOI:
   name: DOI
   display_name: DOI
   description: |
-    Used to specify the locations of all source datasets.
+    Used to specify the locations of all source datasets (BIDS or not).
```

I think such wording is not even correct, 

```suggestion
    Used to specify the Digital Object Identifier (DOI).
```

as a generic thing, I do not think we should associate it here in documentation with source datasets... and it is not about location per se!

---

If it's a term, we might as well define it.

```suggestion
    A Digital Object Identifier (DOI) is a persistent, resolvable name for a digital resource.
```